src/lisp/kernel/clos/print.lisp

```diff
@@ -332,47 +349,15 @@ printer and we should rather use MAKE-LOAD-FORM."
 ;;; ----------------------------------------------------------------------
 
 (defmethod describe-object ((obj t) (stream t))
```

Comment:
Might be better to have this method be on `standard-object` or something instead. We probably have methods for built in classes already, but if not things could get a little weird using `class-slots` and all

Reply:
Sounds good. BTW, we don't really have any describe-object methods right now. Most all is handled in the pre-CLOS inspect.lisp